qa/supply-chain/config.toml

```diff
@@ -7,18 +7,45 @@ version = "0.6"
 [imports.bytecode-alliance]
 url = "https://raw.githubusercontent.com/bytecodealliance/wasmtime/main/supply-chain/audits.toml"
 
-[imports.chromeos]
-url = "https://chromium.googlesource.com/chromiumos/third_party/rust_crates/+/main/cargo-vet/audits.toml?format=TEXT"
-
 [imports.embark-studios]
 url = "https://raw.githubusercontent.com/EmbarkStudios/rust-ecosystem/main/audits.toml"
 
+[imports.google]
+url = "https://raw.githubusercontent.com/google/supply-chain/main/audits.toml"
+
 [imports.isrg]
 url = "https://raw.githubusercontent.com/divviup/libprio-rs/main/supply-chain/audits.toml"
 
 [imports.mozilla]
 url = "https://raw.githubusercontent.com/mozilla/supply-chain/main/audits.toml"
 
+[policy.equihash]
+audit-as-crates-io = false
+
+[policy.f4jumble]
+audit-as-crates-io = false
+
+[policy.orchard]
+audit-as-crates-io = false
```

Comment:
What happens when we remove the patch entries but forget to remove `audit-as-crates-io` from here? Ideally something will break.

Reply:
`cargo-vet` will stop working again until the relevant lines are removed. Case-in-point, the next PR I make will be to migrate to `orchard 0.4`, and I get this error:
```
$ cargo vet
ERROR 
  × There are some issues with your policy.audit-as-crates-io entries

Error: 
  × some audit-as-crates-io policies don't match first-party crates
  │   orchard
  help: Remove the audit-as-crates-io entries
```